README.adoc

```diff
@@ -0,0 +1,63 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+[[introduction-addons]]
+== Introduction to add-ons
+
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Minishift add-ons] are extensions, or plug-ins, that customize the default OpenShift cluster configuration and behavior.
+For example, you can use an add-on to allow authenticated users to run images without a pre-allocated UID, or to run commands on the Docker daemon inside the Minishift VM.
```

Comment:
This are actually two examples from different levels here:

"you can use an add-on to allow authenticated users to run images without a pre-allocated UID" - this is an actual add-on, the anyuid add-on. 

"run commands on the Docker daemon inside the Minishift VM" - this is an example of a specific type of add-on DSL command. 

This are apple and pears. To stay on the same level:

"For example, you can use an add-on to allow authenticated users to run images without a pre-allocated UID (the [anyuid](https://github.com/minishift/minishift/tree/master/addons/anyuid) add-on), or to expose the OpenShift registry as a route ([registry-route](https://github.com/minishift/minishift/tree/master/addons/registry-route) add-on)."